src/utils.rs

```diff
@@ -118,53 +135,69 @@ pub fn check_for_config() -> Result<bool, anyhow::Error> {
 
     if !bedrust_config_file_path.exists() || !model_config_file_path.exists() {
         Ok(false)
-    } else { 
+    } else {
         Ok(true)
     }
+}
+
+pub fn prompt_for_model_selection() -> Result<ArgModels, anyhow::Error> {
+    let model_list = ArgModels::value_variants();
+    let idx = FuzzySelect::with_theme(&ColorfulTheme::default())
+        .with_prompt("Select a default model to use or <enter> to skip")
```

Comment:
This should be just `Select a model to use` (since this is the non-optional version of the selector)